idb/idb.go

```diff
@@ -452,6 +456,7 @@ type AppReverseDelta struct {
 	ClearStateProgram []byte                 `codec:"clearp,omitempty"`
 	LocalStateSchema  sdk_types.StateSchema  `codec:"lsch,omitempty"`
 	GlobalStateSchema sdk_types.StateSchema  `codec:"gsch,omitempty"`
+	ExtraProgramPages uint32                 `codec:"epp,omitempty"`
```

Comment:
We should probably set this field in `commitRoundAccounting()` or remove it. The situation with `AppReverseDelta` is strange. It's not used, not tested, and `LocalStateSchema` and `GlobalStateSchema` fields are completely unused. I really want to remove it. @winder what do you think about this?

Reply:
It is set https://github.com/algorand/indexer/pull/545/files#diff-8fbec00be6e6b2f77a294cbe9a3ee8e3fbdcb5eb13be3046e5da8277302264c2R1140

Reply:
That's `AppParams`, not `AppReverseDelta`. Or am I missing something?

Reply:
heh, I got confused. Added assignment to `reverseDelta` as well.